src/tests/ftest/deployment/network_failure.py

```diff
@@ -448,11 +448,20 @@ def test_network_failure_isolation(self):
             self.log.debug("## Call %s on %s", command, self.network_down_host)
             time.sleep(20)
 
-        # Some ranks may be excluded after bringing up the network interface, so wait
-        # until they are up (joined).
-        if not self.wait_for_ranks_to_join():
-            msg = "One or more servers crashed after bringing up the network interface!"
-            errors.append(msg)
+        # Some ranks may be excluded after bringing up the network interface. Check if
+        # all ranks are joined. If not, restart the servers and check again.
+        time.sleep(60)
```

Comment:
We do not want to use large sleeps in our test code.  Instead, we should query at a much smaller interval with an overall timeout.   This way we don't waste cycles when the test could be progressing.  Something similar to https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/command_utils.py#L711-L726 except we issue `check_system_query_status(dmg_command.system_query())` in the while loop with a `sleep(1)` if not all ranks are joined.

Reply:
Done. Thanks.